src/renderer/kube-watch-api/kube-watch-api.ts

```diff
@@ -96,7 +96,7 @@ export class KubeWatchApi {
 
     const loadThenSubscribe = async (namespaces: string[]) => {
       try {
-        await store.loadAll({ namespaces, reqInit: { signal: childController.signal }, onLoadFailure });
```

Comment:
Could we somehow have a unit test to cover this case?

I am think something along the lines of:
1. Add items to store manually
2. call `.subscribe()`
3. Ensure than old items no longer in store.